test/tests/menubar_menubar-navigation.js

```diff
@@ -143,22 +143,37 @@ ariaTest(
 );
 
 ariaTest(
-  'Test for role="menuitem" in menubar',
+  'Test roving tabindex',
   exampleFile,
-  'menuitem-role',
+  'menuitem-tabindex',
   async (t) => {
-    const menuitems = await t.context.queryElements(
-      t,
-      ex.menubarMenuitemSelector
-    );
+    // Wait for roving tabindex to be initialized by the javascript
+    await exampleInitialized(t);
 
+    await assertRovingTabindex(t, ex.menubarMenuitemSelector, Key.ARROW_RIGHT);
+  }
+);
+
+/*
```

Comment:
This comments out most 90% of the tests

Reply:
delete the commented code

Reply:
Think this comment was for the other line